### ISSUE TITLE
Domemtech.Trash doesn't build because project was not found

### ISSUE THREAD

**Issue by fabriciofx** [kaby76/Domemtech.Trash#130]
@kaby76  I can't compile the Domemtech.Trash project. I'm using:

```
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04 LTS
Release:        22.04
Codename:       jammy
```
and
```
.NET SDK (reflecting any global.json):
 Version:   6.0.301
 Commit:    43f9b18481

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  22.04
 OS Platform: Linux
 RID:         ubuntu.22.04-x64
 Base Path:   /usr/share/dotnet/sdk/6.0.301/

Host (useful for support):
  Version: 6.0.6
  Commit:  7cca709db2

.NET SDKs installed:
  6.0.301 [/usr/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.6 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.6 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```

Steps to reproduce (bug?):

1. Download project files
```
$ git clone https://github.com/kaby76/Domemtech.Trash.git
```

2. Build project
```
$  make
rm -rf */bin */obj
dotnet restore --ignore-failed-sources
/usr/share/dotnet/sdk/6.0.301/NuGet.targets(289,5): error MSB3202: The project file "/mnt/c/Users/fbc/code/Domemtech.Globbing/Domemtech.Globbing.csproj" was not found. [/mnt/c/Users/fbc/code/Domemtech.Trash/Trash.sln]
make: *** [makefile:3: build] Error 1
```

**Comment by kaby76** [comment]
> $ git clone https://github.com/kaby76/Domemtech.Trash.git

No, don't do that. Try `dotnet tool install -g trgen`. It's all there in nuget.

**Comment by fabriciofx** [comment]
@kaby76 It's worked! Thanks! But it need generate the files inside the antlr grammar repository as you did before. Is there a way to do is only *.g4 grammar files? If not, can you update the doc for it? Thanks!

**Comment by kaby76** [comment]
Trgen looks for a pom.xml file in the directory the app is invoked. If there is no pom.xml, try "trgen -s start-symbol-name". It looks at all .g4 files, parsing everyone and builds up dependencies between each, i.e. parser grammars that depend on lexer grammars, and import statements. If the grammars don't parse, nothing can be generated.

**Comment by kaby76** [comment]
The build issue will be fixed in 0.16.6. TrashBase has been merged back into Trash, and it's now being tested via Github Actions, e.g., https://github.com/kaby76/Domemtech.Trash/runs/7950181883?check_suite_focus=true
